dotnet/src/InternalUtilities/src/Diagnostics/ModelDiagnostics.cs

```diff
@@ -21,6 +21,7 @@ namespace Microsoft.SemanticKernel.Diagnostics;
 ///    `SEMANTICKERNEL_EXPERIMENTAL_GENAI_ENABLE_OTEL_DIAGNOSTICS`
 ///    `SEMANTICKERNEL_EXPERIMENTAL_GENAI_ENABLE_OTEL_DIAGNOSTICS_SENSITIVE`
 /// </summary>
+[Experimental("SKEXP0001")]
```

Comment:
This class is using experimental features and is included in the vector store projects via internal utilities.
Since the projects are not experimental anymore, this class showed up with a warning saying that this is using experimental features in a non-experimental project.
Adding experimental here to remove the warning, without having to suppress the warning at project level.